metricbeat/mb/mb.go

```diff
@@ -57,6 +57,7 @@ type MetricSet interface {
 	Host() string   // Host returns a hostname or other module specific value
 	// that identifies a specific host or service instance from which to collect
 	// metrics.
+	Raw() bool // Raw defines if raw metrics should be sent
```

Comment:
I'd like to avoid adding new methods to this interface were possible. In this case, when you need this information it is available in the MetricSet by `m.Module().Config().Raw`.